scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl

```diff
@@ -386,30 +381,30 @@ task ModelSegments {
         GATK_JAR=${default="/root/gatk.jar" gatk4_jar_override}
 
         java -Xmx${command_mem}m -jar $GATK_JAR ModelSegments \
-            --denoisedCopyRatios ${denoised_copy_ratios} \
-            --allelicCounts ${allelic_counts} \
-            ${"--normalAllelicCounts " + normal_allelic_counts} \
-            --maxNumSegmentsPerChromosome ${default="500" max_num_segments_per_chromosome} \
-            --minTotalAlleleCount ${default="30" min_total_allele_count} \
-            --genotypingHomozygousLogRatioThreshold ${default="-10.0" genotyping_homozygous_log_ratio_threshold} \
-            --genotypingBaseErrorRate ${default="0.05" genotyping_base_error_rate} \
-            --kernelVarianceCopyRatio ${default="0.0" kernel_variance_copy_ratio} \
-            --kernelVarianceAlleleFraction ${default="0.025" kernel_variance_allele_fraction} \
-            --kernelScalingAlleleFraction ${default="1.0" kernel_scaling_allele_fraction} \
-            --kernelApproximationDimension ${default="100" kernel_approximation_dimension} \
-            --windowSize ${sep= " --windowSize " window_sizes} \
-            --numChangepointsPenaltyFactor ${default="1.0" num_changepoints_penalty_factor} \
-            --minorAlleleFractionPriorAlpha ${default="25.0" minor_allele_fraction_prior_alpha} \
-            --numSamplesCopyRatio ${default=100 num_samples_copy_ratio} \
-            --numBurnInCopyRatio ${default=50 num_burn_in_copy_ratio} \
-            --numSamplesAlleleFraction ${default=100 num_samples_allele_fraction} \
-            --numBurnInAlleleFraction ${default=50 num_burn_in_allele_fraction} \
-            --smoothingThresholdCopyRatio ${default="2.0" smoothing_threshold_copy_ratio} \
-            --smoothingThresholdAlleleFraction ${default="2.0" smoothing_threshold_allele_fraction} \
-            --maxNumSmoothingIterations ${default=10 max_num_smoothing_iterations} \
-            --numSmoothingIterationsPerFit ${default=0 num_smoothing_iterations_per_fit} \
+            --denoised-copy-ratios ${denoised_copy_ratios} \
+            --allelic-counts ${allelic_counts} \
+            ${"--normal-allelic-counts " + normal_allelic_counts} \
+            --minimum-total-allele-count ${default="30" min_total_allele_count} \
+            --genotyping-homozygous-log-ratio-threshold ${default="-10.0" genotyping_homozygous_log_ratio_threshold} \
+            --genotyping-base-error-rate ${default="0.05" genotyping_base_error_rate} \
+            --maximum-number-of-segments-per-chromosome ${default="500" max_num_segments_per_chromosome} \
```

Comment:
The default value for maximum-number-of-segments-per-chromosome here needs to be synced with the default in the jar.